api/src/main/proto/bio/terra/tanagra/underlay/mapping.proto

```diff
@@ -51,9 +51,23 @@ message AttributeMapping {
     ColumnId column_id = 1;
   }
 
+  // A mapping where an attribute refers to a normalized column in a fact table
```

Comment:
Can you expand on the terminology "fact" here? I'm not quite following this. What makes this a fact, vs just a related table?

Reply:
"fact" probably wasn't right. How about "lookup"? Tried to add some more comments, PTAL.